### PR TITLE
Update GCP DM docs for istio08 release

### PR DESCRIPTION
**Pull request by salrashid123**
Update GCP DM docs for ```0.8.0``` (mostly just the screenshot and services that now show up (eg. ingressgateway)

ref, the actual changes to the template are on   https://github.com/istio/istio/pull/6010

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 73f1ec68dca8924a562d1ddafaffea7100c0d7ef

https://app.netlify.com/sites/preliminary-istio/deploys/5b155cd111b73b061cd88938

File: content/docs/setup/kubernetes/quick-start-gke-dm.md [hunk 1]
```diff
@@ -70,9 +70,9 @@ Once deployment is complete, do the following on the workstation where you've in
 running and Istio is enabled
 
     ```command
-    $ gcloud container clusters list
-    NAME           ZONE           MASTER_VERSION                    MASTER_IP       MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
-    istio-cluster  us-central1-a  v1.9.2-gke.1                      130.211.216.64  n1-standard-2  v1.9.2-gke.1  3          RUNNING
+    gcloud container clusters list
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start-gke-dm.md]: Add leading $ on the command.

File: content/docs/setup/kubernetes/quick-start-gke-dm.md [hunk 2]
```diff
@@ -88,40 +88,48 @@ running and Istio is enabled
...
-deploy/grafana             1         1         1            1           3m
-deploy/istio-citadel       1         1         1            1           3m
-deploy/istio-ingress       1         1         1            1           3m
-deploy/istio-initializer   1         1         1            1           3m
-deploy/istio-mixer         1         1         1            1           3m
-deploy/istio-pilot         1         1         1            1           3m
-deploy/prometheus          1         1         1            1           3m
-deploy/servicegraph        1         1         1            1           3m
-deploy/zipkin              1         1         1            1           3m
+  kubectl get deployments,ing -n istio-system
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start-gke-dm.md]: Add $

File: content/docs/setup/kubernetes/quick-start-gke-dm.md [hunk 3]
```diff
@@ -88,40 +88,48 @@ running and Istio is enabled
...
+deploy/productpage-v1   1         1         1            1           7m
+deploy/ratings-v1       1         1         1            1           7m
+deploy/reviews-v1       1         1         1            1           7m
+deploy/reviews-v2       1         1         1            1           7m
+deploy/reviews-v3       1         1         1            1           7m
+```
+
+Now get the ```istio-ingressgateway``` IP:
+
+```
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start-gke-dm.md]: Should be ```command

File: content/docs/setup/kubernetes/quick-start-gke-dm.md [hunk 4]
```diff
@@ -130,9 +138,10 @@ You can also view the installation using the ***Kubernetes Engine -> Workloads**
 
 1.  Set up an environment variable for Bookinfo's external IP address:
 
-    ```command
-    $ kubectl get ingress -o wide
-    $ export GATEWAY_URL=35.202.120.89
+    ```bash
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start-gke-dm.md]: Should be ```command
Please add $ in front of all commands.

File: content/docs/setup/kubernetes/quick-start-gke-dm.md [hunk 5]
```diff
@@ -144,7 +153,7 @@ You can also view the installation using the ***Kubernetes Engine -> Workloads**
 
 1.  Now send some traffic to it:
 
-    ```command
+    ```bash
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start-gke-dm.md]: Change back to ```command.

File: content/docs/setup/kubernetes/quick-start-gke-dm.md [hunk 6]
```diff
@@ -88,40 +88,48 @@ running and Istio is enabled
...
-deploy/grafana             1         1         1            1           3m
-deploy/istio-citadel       1         1         1            1           3m
-deploy/istio-ingress       1         1         1            1           3m
-deploy/istio-initializer   1         1         1            1           3m
-deploy/istio-mixer         1         1         1            1           3m
-deploy/istio-pilot         1         1         1            1           3m
-deploy/prometheus          1         1         1            1           3m
-deploy/servicegraph        1         1         1            1           3m
-deploy/zipkin              1         1         1            1           3m
+$  kubectl get deployments,ing -n istio-system
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start-gke-dm.md]: Just one space after the $

Comment (salrashid123) [reply content/docs/setup/kubernetes/quick-start-gke-dm.md]: done; also addressed previous comments

**Pull request merged**